build.sbt

```diff
@@ -112,11 +112,32 @@ lazy val http4s = project
   )
   .dependsOn(coreJVM)
 
+lazy val akkaHttp = project
+  .in(file("akka-http"))
+  .settings(name := "caliban-akka-http")
+  .settings(commonSettings)
+  .settings(
+    libraryDependencies ++= Seq(
+      "dev.zio"           %% "zio-interop-cats" % "2.0.0.0-RC10",
```

Comment:
I think the dependencies to `zio-interop-cats` and `cats-effect` are not necessary for this adapter.

How about `circe-generic`? caliban core contains instances of circe `Encoder` and `Decoder` already so we shouldn't need it?

Reply:
resolved in 3336d63